src/win/util.c

```diff
@@ -1744,10 +1744,13 @@ int uv_gettimeofday(uv_timeval64_t* tv) {
   return 0;
 }
 
-int uv__random_rtlgenrandom(void* buf, size_t buflen) {
+int uv__random_winrandom(void* buf, size_t buflen) {
   if (buflen == 0)
     return 0;
 
+  if (pProcessPrng != NULL && pProcessPrng(buf, buflen))
+    return 0;
+
   if (SystemFunction036(buf, buflen) == FALSE)
```

Comment:
Thoughts on removing the RtlGenRandom fallback? ProcessPrng is Windows 10-only (which is fine) and at least Wine 9.0 has it but I don't know about older Wine versions (and I also don't know if we care about older versions.)

I think but am not 100% sure it also exists in Wine 8.

Reply:
> I think but am not 100% sure it also exists in Wine 8.

Added in Wine 8.13 https://github.com/wine-mirror/wine/commit/83d4075202ea595f6beb4a39ced54dbabc6a2e21 - probably a little too recent to fully remove the fallback

Reply:
Thanks, Keno. We'll keep it around for a while then.